CHANGES.md

```diff
@@ -6,9 +6,10 @@ Changes
 
 ### Unreleased
 
+* 2023-11-04 - Bugfix: Pass footer content without text_to_html div generation, solves #442.
```

Comment:
Just to be clear: This bugfix is about the _footnote_, not the _footer_. Would you mind to fix this in the changelog?